build/chip/java/kotlinc_runner.py

```diff
@@ -127,7 +127,7 @@ def main():
     if classpath:
         kotlin_args += ["-classpath", classpath]
 
-    kotlin_args += ["-J-Xms256m", "-J-Xmx4096m", "-J-XX:MaxPermSize=350m",
+    kotlin_args += ["-J-Xms256m", "-J-Xmx4096m", "-J-XX:ReservedCodeCacheSize=350m",
```

Comment:
Why we set  ReservedCodeCacheSize twice?

Reply:
Oh.. my mistake.. Removed. Thanks.